systers_portal/blog/forms.py~

```diff
@@ -0,0 +1,104 @@
+from common.forms import ModelFormWithHelper
```

Comment:
Look at the name of the file. It's `forms.py~`. I assume your text editor created a backup file. You shouldn't commit those. And never do `git add .`. It leads to unwanted code in the commit.